cluster/warewulf-cluster.spec.in

```diff
@@ -12,6 +12,8 @@ URL: http://warewulf.lbl.gov/
 Source: %{name}-%{version}.tar.gz
 ExclusiveOS: linux
 Requires: warewulf-common warewulf-provision
+Requires: nfs-utils
+Requires: ntp
```

Comment:
Don't believe "ntp" is available in CentOS8 or Suse15. Reasonable likely that Chrony will be installed on these distros by default even in minimal installs.

Reply:
Opened #265 to add nfs-utils and tcpdump to development.